### DISCUSSION TITLE
S3 Multipart Downloads?

### DISCUSSION THREAD

**Discussion by StraightfaceStudios** [iterate-ch/cyberduck#14427] [Q&A]
The docs refer to S3 Multipart Uploads, but not downloads.  Is there a way to enable multipart downloads as well as uploads? If I download a 10GB file I don't see the speed change in cyberduck regardless of connection count.  (~300Mbps over 1,000Mbps WAN)

However, if I download three 10GB files simultaneously they individually use the same bandwidth and saturate the link.

**Comment by dkocher** [comment]
This is supported and labeled _Segmented Downloads_ [^1] configurable in Cyberduck and enabled by default when choosing to _Keep Offline_ files with _Smart Synchronization_ connect mode in Mountain Duck.

[^1]: https://docs.cyberduck.io/cyberduck/download/#segmented-downloads
